document.go

```diff
@@ -83,6 +83,10 @@ type GeoShapeField interface {
 	GeoShape() (GeoJSON, error)
 }
 
+type DenseVectorField interface {
+	DenseVector() ([]float64, int, string)
```

Comment:
The vector returned should be of type float32. Also, I'm thinking if its better to split up the APIs to following:
```
DenseVector() []float32
Dims() int
Similarity string
```
That way the caller of the specific APIs get only what they need. Thoughts?

Reply:
Done
Thanks